src/styles/blocks/main.scss

```diff
@@ -0,0 +1,27 @@
+.container {
+  padding: 50px 40px;
+  display: grid;
+  gap: 46px 48px;
+  grid-template-columns: repeat(1, $cardWidth);
```

Comment:
The variable `$cardWidth` is used here, but it's not defined in this file. Ensure that `$cardWidth` is defined elsewhere in your SCSS files or imported correctly.